packages/react-components/src/components/styles/DomainPermissions.styles.ts

```diff
@@ -78,3 +78,15 @@ export const linkTextStyles: ILinkStyles = {
     textAlign: 'inherit'
   }
 };
+
+/**
+ * @internal
+ */
+export const primaryButtonStyles: IButtonStyles = {
```

Comment:
Does this need to set the colour to theme.primary? otherwise it wont change if Contoso sets a custom color no?

Reply:
Not exactly, but this did bring up a good point about being able to style the button accordingly.
I added a style prop so Contoso can change the color of the button and the troubleshooting link as necessary.

For example, Contoso can do something like this now to change the button color:
![image](https://user-images.githubusercontent.com/97130533/193694950-75191caf-b70f-4905-8bf2-38c81dad65ab.png)

https://user-images.githubusercontent.com/97130533/193694748-bb0abee1-661c-4c22-a6c7-0e35e40ade89.mov